spec/models/institution_builder_spec.rb

```diff
@@ -99,6 +99,16 @@
       end
     end
 
+    describe 'when preventing duplicates' do
+      it 'duplicates in WEAMs are not present in institutions' do
+        create :weam, facility_code: '18181818', institution: 'REAL SCHOOL'
+        create :weam, facility_code: '18181818', institution: 'FAKE SCHOOL'
+        described_class.run(user)
+        expect(institutions.last.institution).to eq('REAL SCHOOL')
+        expect(institutions.last.institution).not_to eq('FAKE SCHOOL')
+      end
```

Comment:
Why not a test like? last might not always be reliable
```
        expect(institutions.where(facility_code: '18181818').count).to eq(1)
```

Reply:
I like that a little better, will update.